client/src/components/HistoryViewer/Loading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+/**
+ * Renders a SilverStripe loading animation
+ */
+const Loading = () => (
+  <div className="flexbox-area-grow">
```

Comment:
I'm really surprised this isn't in our library already. Can you have a poke around and see if it seems reasonable to roll into `silverstripe/admin`? Seems like a reasonable add to me.

Reply:
I've raised some pull requests to add this to admin and dogfood it in asset-admin and campaign-admin: https://github.com/silverstripe/silverstripe-admin/pull/490